docs/ssh-keys/deploy-ssh-key.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Deploy SSH Key
+intro: Deploy your SSH Keys to each of the servers you want the keys on.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/ssh-keys/link-ssh-key
+    next:
+        - /docs/servers/ssh-into-server
+    guides:
+    related:
+    featured:
+required_permissions:
+    - application:pipeline_run
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `SSH Keys` card
+1. Click `View` in the `SSH Keys` card to see a list of existing `SSH Keys`
+1. Click `DEPLOY` on the SSH Key you want to deploy
+1. Choose the branch and the pipeline to be deployed, then click `Next`
```

Comment:
"Choose the branch". I think this description have been copied from another doc, right? Because SSH doesn't have "branch".

```suggestion
1. Choose the pipeline to be deployed, then click `Next`
```